packages/common/amplify_db_common/example/lib/db.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Not a major issue, but is there a way to change your commit so that this is a rename?  These files were created by Jordan but are just showing up as new.